mp/src/game/shared/neo/weapons/weapon_aa13.h

```diff
@@ -41,6 +41,8 @@ class CWeaponAA13 : public CNEOBaseCombatWeapon
 	Activity GetPrimaryAttackActivity(void) OVERRIDE;
 
 	virtual float GetFireRate() OVERRIDE { return 0.333f; }
+	
+	virtual const Vector& GetBulletSpread(void) OVERRIDE { return VECTOR_CONE_7DEGREES; };
```

Comment:
This is dangling, because the referenced Vector goes out of scope when this returns. Should be declared as a static variable inside the function.